Makefile

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+install:
+	go install
```

Comment:
This line will generate and install an executable _(go binary)_ named `ali` on your system, and the user will be able to call id directly with `$ ali`.

The problem is we want `ali` to be the alias described on the README, which internally calls go binary, and as they have the same name it will produce conflicts.

On the other hand this command  _make install_ is to provide the user with a something to work on development. So, instead, we should use `go build` that will be generating the binary on the current folder.

Reply:
Ok, I understand that. I'll change it.